### ISSUE TITLE
Ability to share text

### ISSUE THREAD

**Issue by rusberiaosa** [iyox-studios/iyox-Wormhole#23]
Some clients have support to send and receive text snippets. This might be helpful for quickly sharing clipboard contents.

**Comment by EwuUwe** [comment]
I agree, sending text snippets would be quite useful. However, the Rust implementation of the Magic Wormhole, on which this app relies, does unfortunately not yet support text messaging. 
See https://github.com/magic-wormhole/magic-wormhole.rs/issues/131

I will leave this open for now and implement the feature as soon as it's supported by magic-wormhole.rs
